components/Dialogs/examples/DialogsTypicalUseViewController.m

```diff
@@ -72,6 +69,19 @@ - (void)viewDidLoad {
   ]];
 }
 
+- (MDCContainerScheme *)containerScheme {
```

Comment:
This method is named like a getter, but it dispenses a new MDCContainerScheme each time it is called, which is an unexpected outcome.

Reply:
This pattern will be deleted once the app is injecting container schemes.

Reply:
Separately, getter methods don’t always have to have backing by storage.